packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js

```diff
@@ -77,6 +77,59 @@ export default class MeetingInfoV2 {
     });
   }
 
+  /**
+     * Fetches meeting info from the server
+     * @param {String} conversationUrl one of many different types of destinations to look up info for
+     * @returns {Promise} returns a meeting info object
+     * @public
+     * @memberof MeetingInfo
+     */
+  async createAdocSpaceMeeting(conversationUrl) {
```

Comment:
I think there is one typo mistake here. The function name should be createAdhocSpaceMeeting.